zypper_test.go

```diff
@@ -43,8 +46,8 @@ func TestIsZypperExitCodeSevere(t *testing.T) {
 		zypperExitNoRepos,
 		zypperExitZyppLocked,
 		zypperExitErrCommit,
-		zypperExitIndCapNotFound,
```

Comment:
I will make this clear in the commit message and the PR. IMHO this was 'bug' before as zypper doesn't will still continue after a non-existent package has been found.